spec/integration/rails/app.rb

```diff
@@ -14,32 +14,25 @@ class Application < ::Rails::Application
   end
 end
 
-LAYOUT = <<-HTML.freeze
+PAGE = <<-HTML.freeze
 <!DOCTYPE html>
 <html>
 <head>
   <title>TestApp</title>
   <%= csrf_meta_tags %>
 </head>
 <body>
-<%= yield %>
+  <h1>Hello, world!</h1>
 </body>
 </html>
 HTML
 
-INDEX = '<h1>Hello, world!</h1>'.freeze
-
 class ApplicationController < ActionController::Base
   include Rails.application.routes.url_helpers
 
-  layout 'application'
-
-  self.view_paths = [ActionView::FixtureResolver.new(
-    'layouts/application.html.erb' => LAYOUT,
-    'application/index.html.erb'   => INDEX
-  )]
-
-  def index; end
+  def index
+    render inline: PAGE
```

Comment:
🤔 Hmm. I think I'm good with this. I don't believe we'll ever do anything where we monkey with view paths so the simplification is good.

Reply:
_something_ changed with the rails 6 release that broke this test and I couldn't pinpoint what it was, this was the only semi reasonable way I could get it passing again since I believe that view paths method was deprecated and removed